mission_control/navigator_missions/nav_missions/detect_deliver.py

```diff
@@ -42,95 +42,78 @@ def _bounding_rect(self,points):
 
     @txros.util.cancellableInlineCallbacks
     def set_shape_and_color(self):
+        target = yield self.navigator.mission_params["detect_deliver_target"].get()
+        if target == "BIG":
+            self.target_offset_meters = self.SHAPE_CENTER_TO_BIG_TARGET
+        elif target == "SMALL":
+            self.target_offset_meters = self.SHAPE_CENTER_TO_SMALL_TARGET
         self.Shape = yield self.navigator.mission_params["detect_deliver_shape"].get()
         self.Color = yield self.navigator.mission_params["detect_deliver_color"].get()
-        print "Looking for ", self.Color, " ", self.Shape
+        fprint("Color={} Shape={} Target={}".format(self.Color, self.Shape, target), title="DETECT DELIVER",  msg_color='green')
 
     @txros.util.cancellableInlineCallbacks
     def get_waypoint(self):
         res = yield self.navigator.database_query("shooter")
         if not res.found:
-            print "Waypoint not found in database, exiting"
+            fprint("shooter waypoint not found", title="DETECT DELIVER",  msg_color='red')
             raise Exception('Waypoint not found')
         self.waypoint_res = res
 
     @txros.util.cancellableInlineCallbacks
     def circle_search(self):
```

Comment:
This function will not work anymore, for the same functionality, use `...move.d_circle_point(...)` instead